.env.example

```
@@ -29,9 +29,10 @@ PUBLIC_EXPLORER=https://blockstream.info
 PUBLIC_LIQUID_EXPLORER=https://blockstream.info/liquid
 
 # Breez SDK Esplora Overrides (OPTIONAL)
-# Use the backend proxy so the paid Blockstream key is applied server-side
-VITE_LIQUID_EXPLORER_URL=http://localhost:3119/api/esplora/liquid  # Production: https://your-api.railway.app/api/esplora/liquid
+# Use the backend proxy so the paid Blockstream key is applied server-side.
+# Keep Liquid unset to allow Breez SDK to use its default Liquid explorer (recommended).
 VITE_BITCOIN_EXPLORER_URL=http://localhost:3119/api/esplora/bitcoin  # Production: https://your-api.railway.app/api/esplora/bitcoin
```

dotenv-linter (4.0.0)

[warning] [ValueWithoutQuotes] This value needs to be surrounded in quotes

(ValueWithoutQuotes)